### PR TITLE
Gherkin Scripts Alerts tab on Monitoring page and Filters Drop Down Menu Scenarios - [ODC-3923]

### PR DESCRIPTION
**Pull request by gajanan-more**
Fixes: https://issues.redhat.com/browse/ODC-4041

Problem: Gherkin Scripts Design for Alerts tab on Monitoring epic ODC-3923

Solution: These are the gherkin scripts designed covering all the scenarios related to Alerts tab on Monitoring epic ODC-3923

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+Scenario: Show alert icon on Topology workloads and associated alerts in sidebar in both graph and list view
+    Given user is at Topology page
+    And user has a node with some alerts
+    When user clicks on the alert 
+    Then monitoring tab will get opened on the sidebar
+    And user will see the alert details there
+
+
+@regression, @smoke, @manual
+Scenario: Show firing alert icon on Topology workloads and associated alerts in sidebar in both graph and list view
```
Comment (debsmita1) [inline frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature:27]: You can change this scenario to test the alerts section under Monitoring Tab in the Topology sidepanel for workloads having firing alerts

File: frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    Given user is at Topology page
+    And user has a node with some firing alerts
+    When user clicks on the alert 
+    Then monitoring tab will get opened on the sidebar
+    And user will see that Events section is replaced with Alerts section
+    And user will see the alert details there
```
Comment (debsmita1) [inline frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature:33]: workloads having firing alerts will have an Alerts Section under Monitoring Tab in the Topology Sidepanel. The Alerts will be sorted according to their severity. If there are no firing alerts user will not see the Alerts section. The Events section has been removed.

File: frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature [hunk 3]
```diff
@@ -0,0 +1,92 @@
+Feature: Alert Icon on Topology for Workloads in sidebar and Alerts tab on Monitoring page
+	User should be aware of alerts related to workloads and should see alerts in the sidebar
+
+
+Background:
+    Given user logged into the openshift application
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature:6]: This is not required

Comment (gajanan-more) [reply frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature]: Updated

**Comment by makambalaji** [comment]
Already vikram has provided lgtm, So I am approving it
/lgtm /approve

File: frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    When user clicks on the Notification Silence button
+    Then user will see a Silence for dropdown
+    And user will see silence for 30 minutes item
+    And user will see silence for 1 hour item
+    And user will see silence for 2 hours item
+    And user will see silence for 1 day item
+
+
+@regression, @smoke, @manual
+Scenario: Display expire time of Notification Silence
```
Comment (sanketpathak) [inline frontend/packages/dev-console/integration-tests/features/monitoring/alert-Details-in-Sidebar-ODC-3923.feature:78]: Can this be not automated as we can read the time value?

**Pull request merged**